client/gutenberg/extensions/map/edit.js

```diff
@@ -195,21 +203,37 @@ class MapEdit extends Component {
 				<Spinner />
 			</Placeholder>
 		);
-		const getAPIInstructions = sprintf(
-			"<p>Before you use a map block, you will need to get a key from <a href='%1$s'>Mapbox</a>. You will only have to do this once.</p><p>Go to <a href='%1$s'>Mapbox</a> and either create an account or sign in. Once you sign in, locate and copy the default access token. Finally, paste it into the token field below.</p>",
-			'https://www.mapbox.com'
-		);
 		const placeholderAPIStateFailure = (
 			<Placeholder icon={ settings.icon } label={ __( 'Map' ) } notices={ notices }>
 				<Fragment>
 					<div className="components-placeholder__instructions">
-						<RawHTML>{ getAPIInstructions }</RawHTML>
+						<p>{ __( 'To use the map block, you need an Access Token.' ) }</p>
+						<p>
+							<ExternalLink href="https://www.mapbox.com">
```

Comment:
What if we linked users directly to https://www.mapbox.com/account/ instead. 
They would see a login page asking them to create an account. 
But the access token would be easier to find.

Reply:
@MichaelArestad what do you think?